tikv/mvcc.go

```diff
@@ -618,6 +640,64 @@ func (store *MVCCStore) deleteKeysInBatch(reqCtx *requestCtx, keys [][]byte, bat
 }
 
 func (store *MVCCStore) GC(reqCtx *requestCtx, safePoint uint64) error {
-	// TODO: implement GC in badger.
+	// We use the gcLock to make sure safePoint can only increase.
+	store.gcLock.Lock()
```

Comment:
This is the only place that use gcLock? Why GC is called concurrently? And is it really necessary?

Reply:
I'm not 100% sure there is no concurrently call from TiDB, just in case.